packages/analytics/addon/components/ca-report-preview.js

```diff
@@ -58,4 +61,14 @@ export default class CaReportPreviewComponent extends Component {
     }
     return null;
   }
+
+  @action
+  exportTable() {
+    next(() => {
+      const wb = XLSX.utils.table_to_book(
+        document.getElementById("reports-table"),
+      );
+      XLSX.writeFile(wb, `${new Date().toLocaleDateString()}.xlsx`);
```

Comment:
maybe include `this.args.slug` in the name, seems odd to only have a date

Reply:
Done